docs/fields.asciidoc

```diff
@@ -655,6 +655,14 @@ type: object
 A user-defined mapping of groups of marks. 
 
 
+[float]
+=== `transaction.sampled`
+
+type: boolean
+
+Sampled transactions have spans. Defaults to true
```

Comment:
How about: `If the transaction is sampled, then all available information has been recorded, otherwise information about spans and context are discarded.